bootstrap.sh

```diff
@@ -145,8 +145,8 @@ function rebuild_host_os {
   fi
 }
 
-#  values: darwin,fedora,rhel,ubuntu,debian,opensuse
 # Validate the distribution name, or toss an error
+# values: darwin,fedora,rhel,ubuntu,debian,opensuse
```

Comment:
Since you're changing this line anyway, it'd be nicer to add a space following the commas.